cmd/syft/internal/options/license.go

```diff
@@ -5,7 +5,9 @@ import (
 )
 
 type licenseConfig struct {
+	// Deprecated: please use include-license-content instead
 	IncludeUnknownLicenseContent bool    `yaml:"include-unknown-license-content" json:"include-unknown-license-content" mapstructure:"include-unknown-license-content"`
```

Comment:
we should add a postload check to see if both the legacy and new fields are set, it so error out with a helpful message to the user to migrate to the new option.

Reply:
Yea +1 - I thought the file above would take care of a graceful upgrade of the value, but the new field will never be "". It will always default to `none` which means if someone sets the old value and just takes the default it will swallow their config silently. I'll add the postload.

Reply:
Added a warning in postload to let the user know we are ignoring the deprecated value and to change over to the latest enum